documentation/modules/configuring/con-config-mirrormaker2-replication.adoc

```diff
@@ -72,19 +70,21 @@ With this configuration applied, topics retain their original names.
 
 == Topic configuration synchronization
 
-Topic configuration is automatically synchronized between source and target clusters.
-By synchronizing configuration properties, the need for rebalancing is reduced.
-
-== Data integrity
-
-MirrorMaker 2.0 monitors source topics and propagates any configuration changes to remote topics, checking for and creating missing partitions.
-Only MirrorMaker 2.0 can write to remote topics.
+MirrorMaker 2.0 supports topic configuration synchronization between source and target clusters. 
+You specify source topics in the MirrorMaker 2.0 configuration.
+MirrorMaker 2.0 monitors the source topics.
+MirrorMaker 2.0 detects and propagates changes to the source topics to the remote topics.
+Changes might include automatically creating missing topics and partitions.
 
 == Offset tracking
 MirrorMaker 2.0 tracks offsets for consumer groups using internal topics.
 
-* The `offset-syncs` topic maps the source and target offsets for replicated topic partitions from record metadata
-* The `checkpoints` topic maps the last committed offset in the source and target cluster for replicated topic partitions in each consumer group
+`offset-syncs` topic:: The `offset-syncs` topic maps the source and target offsets for replicated topic partitions from record metadata.
+`checkpoints` topic:: The `checkpoints` topic maps the last committed offset in the source and target cluster for replicated topic partitions in each consumer group.
+
+As they used internally by MirrorMaker 2.0, you do not interact directly with these topics. 
+
+TIP: You can use Kafka's `RemoteClusterUtils.java` utility in your client application to retrieve the remote offsets for a consumer group from the `checkpoints` topic.
```

Comment:
As this is not needed anymore, is calling it a `TIP` appropriatte? I would call it different and explain more when it is useful.

Reply:
I think it's still useful. The sync feature is not enabled by default and `RemoteClusterUtils` has a couple of other features.

Reply:
I'm not saying it is not useful. But I think most users would prefer to use the automatic sync which is also more versatile since it does not rely on Java. So the context and when to use it should be better explained.

Reply:
I've changed to _NOTE_ and moved under Synchronizing consumer group offsets. Not sure if we want to say more about where it's useful (or how it might be)

> NOTE: If you have an application written in Java, you can use the `RemoteClusterUtils.java` utility to synchronize offsets from the application. The utility fetches remote offsets for a consumer group from the `checkpoints` topic.